build.gradle

```diff
@@ -81,11 +89,46 @@ jar {
     }
 }
 
-// This is a hack to disable the java 8 default javadoc lint until we fix the html formatting
 tasks.withType(Javadoc) {
+    // do this for all javadoc tasks, including gatkDoc
+    options.addStringOption('Xdoclint:none')
+}
+
+javadoc {
+    // This is a hack to disable the java 8 default javadoc lint until we fix the html formatting
+    // We only want to do this for the javadoc task, not gatkDoc
     options.addStringOption('Xdoclint:none', '-quiet')
 }
 
+// Generate Picard Online Doc
+task picardDoc(type: Javadoc, dependsOn: ['cleanPicardDoc', classes]) {
```

Comment:
Ack, I put that in the wrong place, see https://github.com/broadinstitute/barclay/issues/56 instead.